httpie/core.py

```diff
@@ -131,42 +142,69 @@ def program(args, env, log_error):
                 'outfile': env.stdout,
                 'flush': env.stdout_isatty or args.stream
             }
+            if response is final_response:
+                has_branched('program', 6)
+            else:
+                has_branched('program', 7)
             try:
-                if env.is_windows and is_py3 and 'colors' in args.prettify:
-                    write_stream_with_colors_win_py3(**write_stream_kwargs)
+                if env.is_windows:
+                    has_branched('program', 8)
+                    if is_py3:
+                        has_branched('program', 9)
+                        if 'colors' in args.prettify:
+                            has_branched('program', 10)
+                            write_stream_with_colors_win_py3(**write_stream_kwargs)
                 else:
+                    has_branched('program', 11)
```

Comment:
This else will only apply to the first clause being false now. Previously it applied to any of the clauses being false.

Reply:
I think I've missed this myself in some other functions. Since the tests didn't fail. Maybe we can keep it.